CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - On Android, unimplemented events are marked as unhandled on the native event loop.
 - On Windows, added `WindowBuilderExtWindows::with_menu` to set a custom menu at window creation time.
 - On Android, bump `ndk` and `ndk-glue` to 0.3: use predefined constants for event `ident`.
+- On macOS, fix event-closure-captured objects not being dropped on panic.
```

Comment:
```suggestion
- On macOS, fix objects captured by the event loop closure not being dropped on panic.
```
I'm not too sure if this reads better to everyone, but it does to me.

Reply:
Yes, this seems easier to understand.